Makefile

```diff
@@ -72,16 +72,20 @@ test/unittest-d: run-cmake-debug
 	cmake --build dbuild --target unittest -j $(CPU_CORES)
 	pushd dbuild && $(XVFB) tests/unittest/unittest && popd
 
-test/unittest-coverage: run-cmake-coverage
-	cmake --build coverage-build --target unittest -j $(CPU_CORES)
-	pushd coverage-build && $(XVFB) tests/unittest/unittest && popd
 
-coverage-build/foedag.coverage: test/unittest-coverage
-	lcov --no-external --exclude "*_test.cpp" --capture --directory coverage-build/CMakeFiles/foedag.dir --base-directory src --output-file coverage-build/foedag.coverage
+# test/unittest-coverage: run-cmake-coverage
```

Comment:
Why we need this comments?

Reply:
it is an old code, just commented out for now, will delete soon.